src/gui/render.cpp

```diff
@@ -803,28 +803,23 @@ static bool force_no_pixel_doubling = false;
 
 // We double-scan VGA modes and pixel-double all video modes by default unless:
 //
-//  1) Single-scanning or no pixel-doubling is forced in the
-//     configuration
-//  2) Single-scanning or no pixel-doubling is forced by the OpenGL
-//     shader
-//  3) If the interpolation mode is nearest-neighbour
+//  1) Single-scanning or no pixel-doubling is forced by the OpenGL shader
+//  2) If the interpolation mode is nearest-neighbour
```

Comment:
While you're at it, maybe drop the `If`? Otherwise it doesn't quite read right.

Reply:
Good point, fixed.